### PR TITLE
Persistent storage task completed 

### PR DESCRIPTION
**Pull request by mishalfirdous**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,7 +65,33 @@ <h2>Mishal Firdous</h2>
         </div>
       </div>
     </div>
-    <script></script>
+    <script>
+      
+      function displayLocalStorage () {
+        const experienceList = document.getElementById("experienceList");
+        console.log(experienceList);
```
Comment (abdullahshahzad7890) [inline index.html]: We should not push console logs to the main branch or even any branch as it slows down the performance, once we are done with debugging, we should remove them.

File: index.html [hunk 2]
```diff
@@ -61,7 +65,33 @@ <h2>Mishal Firdous</h2>
...
+       const localData =  JSON.parse(localStorage.getItem("experiencedata"));
+       localData.forEach((item)=>{
+       //console.log(item.companyName);
```
Comment (abdullahshahzad7890) [inline index.html]: we should not have unnecessary comments in our code base

File: index.html
Comment (abdullahshahzad7890) [inline index.html:1]: The file is not correctly formatted. There are syntax errors. Please figure them out and fix them.

File: portfolio.html
Comment (abdullahshahzad7890) [inline portfolio.html:1]: we should also display the experiences from the local storage in this page, if I go to the Home and come back here, the experiences will no more be visible here, moreover, this page should be called edit portfolio, while the other one that you are referring as Home should be called as Portfolio.

**Review by abdullahshahzad7890 — CHANGES_REQUESTED** [review]
Please fix the comments.

File: index.html (hunk 1 above)
Comment (mishalfirdous) [reply index.html]: I got it, deleted it from there, and resolved it!

File: index.html (hunk 2 above)
Comment (mishalfirdous) [reply index.html]: resolved!

File: index.html
Comment (mishalfirdous) [reply index.html:1]: resolved!

File: portfolio.html
Comment (mishalfirdous) [reply portfolio.html:1]: resolved!

File: edit portfolio.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+          document.getElementById("companyName").value = "";
+          document.getElementById("startDate").value = "";
+          document.getElementById("endDate").value = "";
+          document.getElementById("description").value = "";
+
+          experienceForm.style.display = "none";
+        });
+      });
+      function displayLocalStorage() {
```
Comment (abdullahshahzad7890) [inline edit portfolio.html]: Name the function as DisplayLocalStorageData or something that makes more sense and move it to separate js file and use it as it is duplicated in the two files.

**Review by abdullahshahzad7890 — CHANGES_REQUESTED** [review]
Waiting for author.

File: edit portfolio.html (hunk 3 above)
Comment (mishalfirdous) [reply edit portfolio.html]: resolved!

**Pull request merged**